src/store/reducers/appReducer.js

```diff
@@ -33,10 +33,27 @@ export const epics = createEpicScenario({
       epics.actions.fetchAzureMapsKey(),
       epics.actions.fetchDeviceGroups(),
       epics.actions.fetchLogo(),
-      epics.actions.fetchReleaseInformation()
+      epics.actions.fetchReleaseInformation(),
+      epics.actions.getSolutionSettings()
     ]
   },
 
+  /** Log diagnostics data */
+  logEvent: {
+    type: 'APP_LOG_EVENT',
+    epic: ({ payload }, store) => {
+      const settings = getDiagnosticsOptOut(store.getState());
+      const diagnosticsOptOut = isDef(settings) ? settings : true;
```

Comment:
The epic shouldn't now about the defined status of the diagnosticsOptOut. The state should always give you the correct value. This is the reason we updated the service call to handle the 404 code internally.

Reply:
done